### ISSUE TITLE
2020 3rd quarter WFMS flags

### ISSUE THREAD

**Issue by wmay** [ASRCsoft/atmoschem.process#108]
- [x] update wind variables
- [x] something wrong with CO spans -- 07-01 to 07-08
- [x] CO not switched back online after last summit session, 07-22 -- also appears to affect NO spans?
- [x] looks like there are some kind of ozone calibrations that need to be flagged -- once every 3 days?
- [x] CO extended off-line for ZA with Pico checks, 09-10
- [x] check particle counter starting from July 9th
- [x] NO span 4.0/4.2 discrepancy
- [x] IPN check?
- [x] check logs for odd cal changes, 08-09 to 08-12

**Comment by RB-ASRC** [comment]
I sent an email regarding the IPN check 06/17/2020:
Hi All,

Joline did a great job with the summit cal checks today, she will have no problem covering for me when I am out next week.

After the cal check we did a moly converter efficiency check with IPN followed by the normal noon NO2 check.  NO and IPN both at 4ppm agreed so the efficiency is still effectively 1.0, see attached.



I sent an email regarding the IPN check 10/7/2020:
Hi Will and Jim,

The summit NOy analyzer needed a calibration today, and I followed it up with a 45 min IPN check at 4ppb. Once it stabilized the 
NOy reported 3.96-4.01 so the converter continues to behave well.

Rich
